sermant-integration-tests/agentcore-test/dynamic-test-first-plugin-v2/pom.xml

```diff
@@ -0,0 +1,47 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>io.sermant.integration</groupId>
+        <artifactId>agentcore-test</artifactId>
+        <version>1.0.0</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>dynamic-test-first-plugin-v2</artifactId>
+
+    <properties>
+        <package.plugin.name>dynamic-test-first-plugin</package.plugin.name>
+        <package.plugin.version>${project.version}</package.plugin.version>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+        <package.plugin.type>plugin</package.plugin.type>
```

Comment:
you can use -D parameter to package a v2 version plugin jar

Reply:
done